layouts/dashboard.tsx

```diff
@@ -1,19 +1,5 @@
-import { Box, Flex } from "@artsy/palette";
-import { BareHeader } from "../components/BareHeader";
-import { Sidebar } from "../components/dashboard/Sidebar";
+import { Layout } from "../pages/dashboard/_layout";
 
-const DashboardLayout = ({ children, ...props }) => {
-  return (
-    <Box my={2} mx={2} height="100%">
-      <BareHeader label={props.title} message={props.message} />
-      <Flex pt={2}>
-        <Sidebar />
-        <Box width="100%" height="100%">
-          {children}
-        </Box>
-      </Flex>
-    </Box>
-  );
+export default (frontMatter) => {
```

Comment:
#minor - named functions produce better stack traces and provide orientation. Over time unnamed functions can leave a dev unfamiliar with code feeling lost, esp if multiple tabs are open.

Reply:
Fair point 👍